tools/msPurity/createMSP.xml

```diff
@@ -106,6 +109,10 @@
             <option value="[M+CH3COO]-" >[M+CH3COO]-</option>
             <option value="[M-H+CH3COOH]-" >[M-H+CH3COOH]-</option>
         </param>
+        <param argument="--include_adducts_custom" type="text" label="Always include the following custom adducts"
```

Comment:
maybe you can add sanatizer/validors here?

Reply:
Good idea. I added a validator https://github.com/computational-metabolomics/mspurity-galaxy/pull/38/files#diff-ece1bde90e87afede1cf59ed850fa09cR19-R21